api/http/handler/kubernetes/ingresses.go

```diff
@@ -577,6 +685,22 @@ func (handler *Handler) deleteKubernetesIngresses(w http.ResponseWriter, r *http
 	return response.Empty(w)
 }
 
+// @id updateKubernetesIngress
+// @summary Update kubernetes ingresses
+// @description Update kubernetes ingresses for the provided environment
+// @description **Access policy**: authenticated
+// @tags kubernetes
+// @security ApiKeyAuth
+// @security jwt
+// @accept json
+// @produce json
+// @param id path int true "Environment (Endpoint) identifier"
+// @param namespace path string true "Environment (Endpoint) identifier"
+// @param body body models.K8sIngressInfo true "Ingress details"
+// @success 200 {string} string "Success"
+// @failure 400 "Invalid request"
+// @failure 500 "Server error"
+// @router /kubernetes/{id}/namespaces/{namespace/ingresses [put]
```

Comment:
'{namespace' typo

Reply:
It might not be the right ticket for it, but is it worth considering changing this path to be `/kubernetes/{id}/namespaces/{namespace}/ingresses/{ingressname} [put]`